mix.exs

```diff
@@ -45,16 +45,16 @@ defmodule ExWebRTC.MixProject do
 
   defp deps do
     [
-      {:ex_sdp, "~> 0.13"},
-      {:ex_ice, "~> 0.1"},
-      {:ex_dtls, "~> 0.14"},
-      {:ex_libsrtp, "~> 0.6"},
-      {:ex_rtp, "~> 0.2"},
-      {:ex_rtcp, "~> 0.1"},
+      {:ex_sdp, "~> 0.13.0"},
+      {:ex_ice, "~> 0.2.0"},
+      {:ex_dtls, "~> 0.14.0"},
+      {:ex_libsrtp, "~> 0.6.0"},
+      {:ex_rtp, "~> 0.2.0"},
+      {:ex_rtcp, "~> 0.1.0"},
```

Comment:
Why add the `path` digit to versions? I hope you don't plan to introduce breaking changes in the libraries with `patch` updates 🙂

Reply:
in packages that are in `0.x.x.`, breaking changes can be introduced in minor